src/frontend/apps/standalone_site/src/features/Playlist/components/PlaylistPage.tsx

```diff
@@ -108,106 +114,134 @@ export const PlaylistPage = () => {
   const shouldDisplayTable = !isError && data && data.count > 0;
 
   return (
-    <Box pad="medium">
-      <WhiteCard direction="column">
-        <Box flex="shrink" direction="row">
-          <Box flex>
-            <Heading level={3} truncate>
-              {intl.formatMessage(messages.title)}
-            </Heading>
-          </Box>
-          {shouldDisplayCreateButton && (
-            <Box flex="shrink" margin={{ vertical: 'auto', left: 'small' }}>
-              <Button primary a11yTitle={intl.formatMessage(messages.create)}>
-                {intl.formatMessage(messages.create)}
-              </Button>
+    <Fragment>
+      <Switch>
+        <Route path={CREATE_PLAYLIST_MODALE}>
```

Comment:
The props `path` in `Route` accepts an array of string, could we centralize the switch in AppRoutes? It gives up an easy understanding of the routemap of the application.
Then to open your modale you can just have a condition checking `useLocation();`.

Reply:
Yeah I thought about it too but I was not a big fan